flask_selfdoc/autodoc.py

```diff
@@ -44,7 +145,10 @@ def init_app(self, app):
         self.add_custom_template_filters(app)
 
     def teardown(self, exception):
-        ctx = stack.top  # noqa: F841
+        if _cv_app is not None:
```

Comment:
I suppose this is some hack to avoid destroying the context or something like that?
I tried to do the same with the new app context object from Flask 2.3.